app/services/workarea/affirm/order.rb

```diff
@@ -101,6 +102,19 @@ def site_name
       def host
         "https://#{Workarea.config.host}"
       end
+
+      # This is used for platform attribution. Please do not change
+      # these values.
+      #
+      # @private
+      # @return [Hash]
+      def metadata
+        {
+          platform_type: 'Workarea',
```

Comment:
The only entry that really matters here is `platform_type: "Workarea"`. The rest of the entries in this metadata hash are just for record-keeping, they don't require any specific format or anything.